frontend/app/utils/locale-utils.ts

```diff
@@ -97,8 +97,9 @@ export function getTypedI18nNamespaces<const T extends Readonly<FlatNamespace>,
  * @param { english translation, french translation}
  * @returns either the english translation or the french translation.
  */
-export function getNameByLanguage(language: string, obj: { nameEn?: string; nameFr?: string }) {
-  return language == 'fr' ? obj.nameFr : obj.nameEn;
+export function getNameByLanguage(language: string, obj: { nameEn?: string; nameFr?: string } | undefined) {
+  if (obj) return language == 'fr' ? obj.nameFr : obj.nameEn;
+  return '';
 }
```

Comment:
I feel the consumer should be handling the case where `obj` is `undefined` (maybe throw a 400 or something).

Reply:
added throw for 400